common/history/characters/0_dvg_timurtash.txt

```diff
@@ -0,0 +1,63 @@
+﻿CHARACTERS = {
```

Comment:
[12:26:08][pdx_persistent_reader.cpp:252]: Error: "Wrong type parsed in create_character effect: Artuqid, near line: 24
Wrong type parsed in create_character effect: =, near line: 24" in file: "common/history/characters/0_dvg_timurtash.txt" near line: 35
[12:26:11][jomini_script_system.cpp:263]: Script system error!
  Error: set_owner_of_provinces effect [ Province 'x809040' crossing multiple state regions ]
  Script location: file: events/dvg_persia.txt line: 127